lang/ilias_de.lang

```diff
@@ -16015,6 +16015,7 @@ tms#:#t_user#:#Ansicht: Mitarbeiter
 tms#:#t_course#:#Ansicht: Kurs
 tms#:#canceling_for#:#Stornierung für %s
 tms#:#duplicate_course_booked#:#Ein Training dieser Vorlage ist bereits gebucht und noch nicht beendet. Sie dürfen dieses Angebot daher nicht buchen. Sie können die Buchungen in der Übersicht ihrer gebuchten Trainings im Cockpit einsehen und dort ggf. Stornierungen vornehmen.
+tms#:#superior_duplicate_course_booked#:#Ein Training dieser Vorlage ist für %s bereits gebucht. Eine buchung dieses Angebots ist daher für Ihn nicht möglich. Sie können seine Buchungen in der Übersicht der buchungen Ihrer Mitarbeiter im Cockpit einsehen und ggf. Stornierugen vornehmen.
```

Comment:
Da bitte nochmal drüber (ist zu groß, sonst hätte ich das online editiert):
superior_duplicate_course_booked#:#Ein Training dieser Vorlage ist für %s bereits gebucht. Eine Buchung dieses Angebots ist daher für diesen Benutzer nicht möglich. Sie können die Buchungen in der Übersicht "Buchungen Ihrer Mitarbeiter" im Cockpit einsehen und ggf. Stornierugen vornehmen.

Reply:
"Buchungen Ihrer Mitarbeiter" muss aber nicht unbedingt so heißen. Das kann ja je Objekt mit dem Titel geändert werden. Deshalb würde ich es gerne neutraler lassen.

Reply:
Sie können die Buchungen Ihrer Mitarbeiter im Cockpit einsehen und ggf. Stornierugen vornehmen.
??

Reply:
Claudia wollte das Wort "Mitarbeiter" nicht, da auch andere die Leute buchen werden, von denen es nicht direkt die Mitarbeiter sind....

Reply:
Eine buchung -> Eine Buchung
ist daher möglich. -> ist daher nicht möglich.
Sie können seine Buchungen -> "seine" ist maskulin, das geht nicht. "Ein Training ist für Heike Schmitt gebucht. Sehen Sie seine Buchungen im Cockpit..." ;((